sass/_Pager.scss

```diff
@@ -28,6 +28,10 @@
   margin-top: 0;
   margin-left: 25px;
   display: inline-block;
+
```

Comment:
I followed the same pattern of adding `coveo-hidden` CSS class to hide a component. However, I wanted to see what your thoughts were on adding a `hide()` and `show()` methods on the BaseComponent class which all components extend. 

The method would hide the component using javascript: `this.element.display = 'none'`

Reply:
Use `Dom.hide()` , `Dom.show()`, which also includes the proper stuff for accessibility.

Reply:
That's really nice. I will need to modify the `.show()` implementation however to set the display to an empty string (i.e. to take on whatever value it had before). At the moment, it sets it to block.